tests/data/test_file.txt.md5

```diff
@@ -0,0 +1 @@
+169293f7c9138e4b50ebcab4358dc509
```

Comment:
Does the current Midas3 support a more secure hash algorithm than MD5? If SHA, etc. is available, the tests should just always use that, since people will likely follow whatever the testing does.

To elaborate, while I agree with @cpatrick that there's a lot of FUD about MD5 floating around, using MD5 to verify external binary data (which may have been initially provided by untrusted third parties) is specifically what it's weak to.